jdl/converters/json-to-jdl-converter.ts

```diff
@@ -108,7 +115,7 @@ function cleanYoRcFileContent(yoRcFileContent) {
   return yoRcFileContent;
 }
 
-function getJSONEntityFiles(applicationDirectory) {
+function getJSONEntityFiles(applicationDirectory: string) {
   const entities = new Map();
```

Comment:
This would make the compiler cry. I think it tolerates not having parameterized types, but displays a warning about it, doesn't it?

We can create an issue to tackle it and use the right types

Reply:
What do you think is wrong with this? TypeScript does not complain about it and it's correct as far as I can tell.

Reply:
Oh? Wow, Last time I checked the compiler threw stones at me.
If there's no issue from the compiler, then let's just create an issue for this.
Having parameterized types, especially in types like the Map or the Set, is useful because TS won't infer the type. It will use "unknown" if memory serves. It's not optimal memory-wise. Plus, having types helps understanding the code

Reply:
I thought you are talking about the function argument while you were talking about the map a line below.

It works because TypeScript infered it as Map<any, any>. I pushed a commit improving the types.